scripts/build-shotcut.sh

```diff
@@ -20,10 +20,13 @@ SOURCE_DIR="$INSTALL_DIR/src"
 ACTION_GET_COMPILE_INSTALL=1
 ACTION_GET_ONLY=0
 ACTION_COMPILE_INSTALL=1
+TAR_APP_PACKAGE=1
+KEEP_BUILD_FILES=0
 SOURCES_CLEAN=1
 INSTALL_AS_ROOT=0
 CREATE_STARTUP_SCRIPT=1
 ENABLE_FREI0R=1
+DISABLE_ALL_EXCEPT_SHOTCUT_AND_MLT=0
```

Comment:
This seems clunky and oddly specific. How about a variable called "BUILD_PROJECTS"? The logic could be:
* If the value is empty, build all projects.
* Otherwise, the variable can contain a list of project names to build.

In the case of how you are using it now, one would specify:
BUILD_PROJECTS="shotcut melt"

This way, If I need to make changes in frei0r, I could specify:
BUILD_PROJECTS="frei0r"
and skip building everything else.

Also, I could specify:
BUILD_PROJECTS="shotcut"
when I am just developing shotcut.